client/layout/poll-invitation/index.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import snakeCase from 'lodash/snakeCase';
+import includes from 'lodash/includes';
+import once from 'lodash/once';
+import debugModule from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import userModule from 'lib/user';
+import { ga as googleAnalytics } from 'analytics';
+import Gridicon from 'components/gridicon';
+import { tracks } from 'analytics';
+import config from 'config';
+import preferencesActions from 'lib/preferences/actions';
+
+/**
+ * Module variables
+ */
+const debug = debugModule( 'calypso:poll-invitation' );
+const user = userModule();
+const _dismissedPreferenceKey = 'dismissedBrazilianSurvey';
+const _sectionWhiteList = [ 'reader', 'sites' ];
+
+function shouldDisplay() {
+	return 'pt-br' === user.get().localeSlug;
+}
+
+function recordEvent( eventAction ) {
+	googleAnalytics.recordEvent( 'Brazil Survey Invitation', eventAction );
+	const tracksEventName = 'calypso_poll_invitation_' + snakeCase( eventAction );
+	debug( 'recording event ' + tracksEventName );
+	tracks.recordEvent( tracksEventName );
+}
+
+const recordDisplayedEventOnce = once( () => recordEvent( 'Displayed' ) );
+
+export default React.createClass( {
+	displayName: 'PollInvitation',
+
+	propTypes: {
+		isVisible: React.PropTypes.bool
```

Comment:
I like where you're coming from, but I'm going to punt this for a separate PR where I can change all three components in `client/layout/index.js` at once.